lib/src/dart.dart

```diff
@@ -112,7 +113,8 @@ class AnalysisContextManager {
       }
 
       var contextBuilder = ContextBuilder();
-      var context = contextBuilder.createContext(contextRoot: root);
+      var context = contextBuilder.createContext(
+          contextRoot: root, sdkPath: getSdkPath());
```

Comment:
`cli_util. getSdkPath()` used to have to do a lot more, but I believe currently it's just a thin wrapper around Platform.resolvedExecutable.

Reply:
Yes. It is just `p.dirname(p.dirname(Platform.resolvedExecutable))` still I think it is nicer to get it from cli_util, so we can update that logic in one place should it change.

Also that logic is actually not enough when you use the binary in `out/ReleaseX64/dart` from a sdk checkout, but I think it is good enough for pub.

Reply:
I would argue that one should never run `dart` binaries from non-SDK, and we should not support this.

As it was explained to me, we sometimes don't have yet all components for full SDK, but I think in this case we should place what we have in the right structure to avoid adding such arbitrary support for non-standard locations.

Reply:
In that case we should probably work on removing those binaries from `out/ReleaseX64` and friends...

Reply:
Please comment on: https://github.com/dart-lang/cli_util/pull/44